eng/testing/tests.mobile.targets

```diff
@@ -33,6 +33,17 @@
     <UseNativeHttpHandler>true</UseNativeHttpHandler>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetOS)' == 'iOS' and '$(UseDefaultiOSFeatureSwitches)' == 'true'">
+  	<EnableUnsafeBinaryFormatterSerialization>false</EnableUnsafeBinaryFormatterSerialization>
+		<EnableUnsafeUTF7Encoding>false</EnableUnsafeUTF7Encoding>
+		<EventSourceSupport>false</EventSourceSupport>
+		<HttpActivityPropagationSupport>false</HttpActivityPropagationSupport>
+		<InvariantGlobalization>true</InvariantGlobalization>
```

Comment:
Is that really true?

@steveisok

Reply:
It shouldn't be the default anymore.

Reply:
Should this property be set  to `false` or removed completely?

Reply:
I would remove it.

Reply:
Removed, thanks